common/read_peer_msg.h

```diff
@@ -57,21 +57,29 @@ bool is_wrong_channel(const u8 *msg, const struct channel_id *expected,
 
 /**
  * handle_peer_gossip_or_error - simple handler for all the above cases.
- * @peer_fd, @gossip_fd: peer and gossip fd.
+ * @peer_fd, @gossip_fd, @gossip_store_fd: peer, gossip and gossip_store fds.
```

Comment:
`channel_id` is missed here...a small corner that originally existed..

Reply:
Yep, indeed.